__tests__/app.test.js

```diff
@@ -160,4 +136,64 @@ describe("app", () => {
 
   });
 
+  //  Will use these tests later for task 17, realized that task 6 only wanted to retrieve all users
+  //  instead of user/user:id
+  //
+  //  describe.only("17. GET /api/users/username", () => {

```

Comment:
this commented out code is not relevant to this pull request, I know you spent time on it but it should really just be removed.